src/server/logging/log_format.js

```diff
@@ -46,10 +46,19 @@ export default class TransformObjStream extends Stream.Transform {
     next();
   }
 
+  formatTimestamp(data, format) {
```

Comment:
__nit/question:__ is there any reason that prevents you from passing just `@timestamp` field instead of entire `data` object. Currently this method has potential access to the info that it shouldn't have + IMO with this method signature it looks more like `extractAndFormatTimestamp` :)

Reply:
I really don't like assuming that the "data" has a `@timestamp` property, which is why formatTimestamp does the property access, it is defined in the same place as the original `@timestamp` property is defined. Switched the method to `extractAndFormatTimestamp()`